S/StarPU/build_tarballs.jl

```diff
@@ -0,0 +1,42 @@
+# Note that this script can accept some limited command-line arguments, run
+# `julia build_tarballs.jl --help` to see a usage message.
+using BinaryBuilder, Pkg
+
+name = "StarPU"
+version = v"1.3.7"
+
+# Collection of sources required to complete build
+sources = [
+    ArchiveSource("https://files.inria.fr/starpu/starpu-1.3.7/starpu-1.3.7.tar.gz", "1d7e01567fbd4a66b7e563626899374735e37883226afb96c8952fea1dab77c2")
+]
+
+# Bash recipe for building across all platforms
+script = raw"""
+cd $WORKSPACE/srcdir
+cd starpu*
+mkdir build
+cd build
+../configure --prefix=${prefix} --build=${MACHTYPE} --host=${target} --enable-fortran
+make -j${nproc}
+make install
+exit
+"""
+
+# These are the platforms we will build for by default, unless further
+# platforms are passed in on the command line
+platforms = expand_gfortran_versions(supported_platforms())
+
+
+# The products that we will ensure are always built
+products = [
+    LibraryProduct("libstarpu", :libstarpu)
```

Comment:
https://docs.binarybuilder.org/dev/reference/#BinaryBuilderBase.LibraryProduct
```julia
julia> using Base.BinaryPlatforms

julia> parse_dl_name_version("libstarpu-1.3.so.5", "linux")[1]
"libstarpu-1.3"
```
```suggestion
    LibraryProduct("libstarpu-1.3", :libstarpu)
```